src/pycolmap/scene/reconstruction.cc

```diff
@@ -249,6 +250,10 @@ void BindReconstruction(py::module& m) {
            [](const Reconstruction& self, const py::dict&) {
              return Reconstruction(self);
            })
+      .def("__assign__",
```

Comment:
Copying the reconstruction on each assignment in Python is not very idiomatic. I suggest to instead create a "copy()" method and leave the `__assign__` to be "copy-by-ref".

Reply:
Indeed, I cannot find a reference to `__assign__` anywhere. Dunder names `__*__` are reserved by the Python language, let's not create a new one please. `pycolmap.Reconstruction` already implements `__copy__` and `__deepcopy__`, I think that an extra method is not needed.

Reply:
I dont think this method is necessary. I will remove it.

Reply:
Ok, I got tricked into thinking that this is the way to overload the assignment operator in Python. Agreed that we shouldn't invent a new `__*__` operator but instead reuse the `__copy__` method or introduce an explicit copy() method. For example, numpy provides a similar method for arrays.